app/src/main/res/values-gl/strings.xml

```diff
@@ -38,171 +37,202 @@
   </plurals>
 
   <plurals name="status_restore_done_details">
-    <item quantity="one">Reestablecido con éxito <xliff:g id="total_items">%1$d</xliff:g> elemento (%2$d dups).</item>
-    <item quantity="other">Reestablecidos con éxito <xliff:g id="total_items">%1$d</xliff:g> elementos (%2$d dups).</item>
+    <item quantity="one">Restaurado con éxito <xliff:g id="total_items">%1$d</xliff:g> elemento (%2$d dups).</item>
+    <item quantity="other">Restaurados con éxito <xliff:g id="total_items">%1$d</xliff:g> elementos (%2$d dups).</item>
   </plurals>
 
-  <string name="status_restore_canceled_details">%1$d/%2$d elementos reestablecidos con éxito.</string>
+  <string name="status_restore_canceled_details">%1$d/%2$d elementos restaurados con éxito.</string>
 
   <string name="status_unknown_error">Erro</string>
   <string name="notification_general_error">Erro de SMSBackup+</string>
 
-  <string name="status_unknown_error_details">Erro durante o gardado/reestablecemento:\n<xliff:g id="error_description">%1$s</xliff:g></string>
+  <string name="status_unknown_error_details">Erro durante o gardado/restauración:\n<xliff:g id="error_description">%1$s</xliff:g></string>
   <string name="status_auth_failure">Erro de inicio de sesión</string>
   <string name="notification_auth_failure">SMSBackup+ erro de inicio de sesión</string>
 
-  <string name="status_auth_failure_details_xoauth">Erro de autorización XOAuth. Por favor, asegúrese de que ten habilitado IMAP na configuración da súa conta de GMail.</string>
+  <string name="status_permission_problem">Erro de permisos</string>
+
+  <plurals name="status_permission_problem_details">
+    <item quantity="one">É necesario o seguinte permiso: %1$s.</item>
+    <item quantity="other">Son necesarios os seguintes permisos: %1$s.</item>
+  </plurals>
+
+  <string name="permission_read_sms">Acceso ás mensaxes SMS</string>
+  <string name="permission_read_call_log">Acceso ao historial de chamadas</string>
+  <string name="permission_read_contacts">Acceso aos contactos</string>
+  <string name="permission_unknown">Descoñecido</string>
+
+  <string name="notification_missing_permission" translatable="false">@string/status_permission_problem</string>
```

Comment:
Thanks for your help! Can you delete the non translatable lines? (`translatable=false`) The build is currently failing for this PR.